dax/controller/controller.go

```diff
@@ -784,6 +784,18 @@ func (c *Controller) Databases(ctx context.Context, orgID dax.OrganizationID, id
 	return c.Schemar.Databases(tx, orgID, ids...)
 }
 
+// get the list of workers being used by a database
+func (c *Controller) GetDatabaseWorkerCount(ctx context.Context, qdbid dax.QualifiedDatabaseID) (int, error) {
```

Comment:
Go convention typically avoids using the term "get" in methods. There is an exception in the handler because that "get" signifies that it's related to the GET http method. This method should be something like `DatabaseWorkerCount()`.

Reply:
Conclusion: just call it `WorkerCount()`